### ISSUE TITLE
Data Split, RNN, Dropout, Rectified Linear Unit

### ISSUE THREAD

**Issue by MrRincon** [MrRincon/ReelFeel_NLP_RNN#37]
- [x] Splitting the Dataset
- [x] Implementing Recurrent Neural Network (RNN)
- [x] Dropout to Prevent Overfitting
- [x] Rectified Linear Unit (ReLU) to Mitigate the Vanishing Gradient Problem
